shakenfist/client/main.py

```diff
@@ -251,6 +254,20 @@ def namespace_show(ctx, namespace=None):
     _show_namespace(ctx, namespace)
 
 
+@namespace.command(name='clean',
+                   help=('Clean (delete) namespace of all instances and networks\n\n'))
```

Comment:
We only do the \n\n at the end of help strings where we then describe arguments.